data/Gibraltar/Parliament/sources/instructions.json

```diff
@@ -0,0 +1,59 @@
+{
+  "sources": [
+    {
+      "file": "morph/data.csv",
+      "create": {
+        "type": "morph",
+        "scraper": "andylolz/gibraltar-parliament",
+        "query": "SELECT * FROM data"
+      },
+      "source": "http://www.parliament.gi/history/composition-of-parliament",
+      "type": "membership"
+    },
+    {
+      "file": "morph/terms.csv",
+      "create": {
+        "file": "morph/terms.csv",
+        "type": "morph",
+        "scraper": "andylolz/gibraltar-parliament",
+        "query": "SELECT * FROM terms"
+      },
+      "source": "http://www.parliament.gi/history/composition-of-parliament",
+      "type": "term",
+      "merge": {
```

Comment:
At the minute there's no way to merge 'term' data — the code will only work with a single 'terms' source. This section will just be ignored, rather than causing any trouble, but it's probably better to remove it just in case something changes later, and this starts doing something unexpected.